lib/phoenix_live_view/test/live_view_test.ex

```diff
@@ -1028,6 +1034,125 @@ defmodule Phoenix.LiveViewTest do
     end
   end
 
+  @doc """
+  Open the default browser to display current HTML of `view_or_element`.
+
+  ## Examples
+
+      view
+      |> element("#term a:first-child()", "Increment")
+      |> open_browser()
+
+      assert view
```

Comment:
```suggestion
      view
```

Reply:
Just to make sure we're on the same page. The second example has an `assert` to show how `open_browser` can be easily added to a "test pipeline", so do you think it's better to remove `assert` and `=~ "Name updated"` at the end?

Reply:
I see, ignore me then but please update the asserts so the pipeline is indented. See the other examples. :)